packages/it/index.js

```diff
@@ -37,23 +37,20 @@ function standardiseDescriptor(descr) {
   var p = isObject(props) || isObject(properties) ?
     assign({}, props, properties) : undefined;
 
-  var dp = isObject(deepProps) ? merge({}, deepProps) : undefined;
-  dp = isObject(deepProperties) ? merge(dp, deepProperties) : dp;
+  var dp = isObject(deepProps) || isObject(deepProperties) ? merge({}, deepProps, deepProperties) : undefined;
```

Comment:
Wouldn't `merge(dst || {}, src)` be more efficient than `merge({}, dst, src)`?

Reply:
The code is a bit different to what wrote.
```js
var dp = isObject(deepProps) || isObject(deepProperties) ? merge({}, deepProps, deepProperties) : undefined;
```
The `dp` must be undefined and both `deepProps` and `deepProperties` are not objects. By specification. Also, the `dp` must be a new object and must not mutate `deepProps` and `deepProperties`.

Reply:
Hum... with es2016 syntax it will be more readable I expect ;)